frontend/src/utils/customHooks/useDisableElementsOnSomeRoutes.js

```diff
@@ -0,0 +1,21 @@
+import { useState, useEffect } from 'react'
+import { useLocation } from 'react-router-dom'
+
+export const useDisableElementsOnSomeRoutes = props => {
+  const location = useLocation()
+  const [disable, setDisable] = useState(false)
```

Comment:
`disable` is boolean. According to convention it should be called `isDisabled`

Reply:
Aye